### PR TITLE
[DOCS] Remove CLI References from GX Core Documentation

### PR DESCRIPTION
**Pull request by kwcanuck**
[DSB-504](https://greatexpectations.atlassian.net/browse/DSB-504) requests the removal of the remaining CLI references in the GX Core documentation. This PR implements this change. 

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Appropriate tests and docs have been updated

[DSB-504]: https://greatexpectations.atlassian.net/browse/DSB-504?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by cdkini — APPROVED** [review]

File: docs/docusaurus/docs/guides/connecting_to_your_data/components/where_to_run_code.md
Comment (Rachel-Reverie) [inline docs/docusaurus/docs/guides/connecting_to_your_data/components/where_to_run_code.md:12]: I think we should update these labels.  Maybe something like: "file editing" and "python" -- or even remove the "file editing" option, since I believe we're trying to get people away from directly editing yaml.

**Review by Rachel-Reverie — APPROVED** [review]
LGTM.  One minor suggestion, and I'm approving so you aren't stuck on code owner approval if you decide to make that change.  But as long as the netlify preview builds without errors we should be good to merge.

**Pull request merged**
